runtime/src/accounts_db.rs

```diff
@@ -1895,6 +1908,17 @@ impl AccountsDb {
         new
     }
 
+    fn init_gradual_filler_accounts(&self, slots_per_epoch: Slot) {
+        let count = self.filler_accounts_config.count;
+        if count > 0 {
+            let accounts_per_slot = (count as u64) / slots_per_epoch;
```

Comment:
Is count guaranteed to be dividable by slots_per_epoch? Do we need to take care of the reminder?

Reply:
no - this is a debug only feature. this is good enough. I can add a comment.